usage-guides/assets.md

```diff
@@ -17,6 +17,28 @@ To solve this problem we added a whitelist which you can add assets to as can be
 
 ![Whitelist asset](/images/asset_whitelist_menu.png)
 
+## Whitelisting and Re-detecting Missing Tokens
+
+If a token doesn't appear in your balances:
+
+### Check Asset Status
+
+Go to the Manage Assets -> Assets page
+Use the search to find the token by name or address
+Verify if the token is ignored
+If ignored, click to whitelist it
+
+![Example of managing assets and whitelisting a token in the Manage Assets page.](/images/manage-assets-refresh.png)
+
+### Re-detect After Whitelisting
+
+Go to Account & Balances -> Blockchain Balances
+Scroll down to EVM accounts and click on Re-detect tokens (🔃) Here you can specify chains for re-detection.
+![Example of re-detecting tokens for all EVM chains in the Blockchain Balances section.](/images/redetect-tokens-chain.png)
+
+If still not appearing, try the full token detection from the dashboard by clicking on 🔃 toggle.
+![Example of performing a full token detection using the re-detect toggle on the dashboard.](/images/redetect-tokens-dashboard.png)
```

Comment:
Here you should specify the different types of detection. The best way to detect tokens for a single address is go to blockchain & accounts balances, go to the address, go the the per chain view and click detect tokens there.

```suggestion
Go to Account & Balances -> Blockchain Balances. Scroll down to EVM accounts and click on Re-detect tokens (🔃) Here you can specify chains for re-detection. ![Example of re-detecting tokens for all EVM chains in the Blockchain Balances section.](/images/redetect-tokens-chain.png)

If still not appearing, try the full token detection from the dashboard by clicking on 🔃 toggle. ![Example of performing a full token detection using the re-detect toggle on the dashboard.](/images/redetect-tokens-dashboard.png)
```